xbmc/music/windows/MusicFileItemListModifier.cpp

```diff
@@ -58,8 +58,8 @@ void CMusicFileItemListModifier::AddQueuingFolder(CFileItemList& items)
 
   auto nodeChildType = directoryNode->GetChildType();
 
-  // No need for "all" when overview node and child node albums or artists
-  if (directoryNode->GetType() == NODE_TYPE_OVERVIEW &&
+  // No need for "all" when overview node and child node albums or artists without options
+  if (directoryNode->GetType() == NODE_TYPE_OVERVIEW && musicUrl.GetOptions().empty() &&
```

Comment:
Again this area is changed is in PR16626, so no need here. Watch out when rebasing!